src/dmd/backend/cod3.c

```diff
@@ -12,6 +12,7 @@
 
 #if !SPP
 
+#include        <climits>
```

Comment:
Pleased don't use the `.h`-less versions. `limits.h` will do fine. Also, it's unnecessary due to below suggestion.